src/web100srv_unit_tests.c

```diff
@@ -47,47 +68,153 @@ void test_e2e() {
   waitpid(server_pid, &server_exit_code, 0);
 }
 
+char* nodejs_command() {
```

Comment:
We're returning temporaries here. I know it's test code but I'm worried about this causing random test failures and being really hard to debug. Can we just make the `char*` a param and populate that?